apps/prairielearn/src/ee/pages/institutionAdminCourse/institutionAdminCourse.sql

```diff
@@ -10,12 +10,31 @@ WHERE
 
 -- BLOCK select_course_instances
 SELECT
-  ci.*
+  to_jsonb(ci.*) AS course_instance,
+  COALESCE(COUNT(e.id), 0)::integer AS enrollment_count
 FROM
   course_instances AS ci
   JOIN pl_courses AS c ON (ci.course_id = c.id)
+  LEFT JOIN enrollments AS e ON (ci.id = e.course_instance_id)
+  LEFT JOIN course_instance_access_rules AS ar ON (ci.id = ar.course_instance_id)
```

Comment:
Nit for consistency:
```suggestion
  LEFT JOIN course_instance_access_rules AS ciar ON (ci.id = ciar.course_instance_id)
```

Reply:
Resolved by bbe856a.